qsirecon/interfaces/mrtrix.py

```diff
@@ -391,6 +401,36 @@ def _list_outputs(self):
         if self.inputs.algorithm in ("msmt_csd", "ss3t"):
             outputs["gm_odf"] = op.abspath(self._gen_filename("gm_odf"))
             outputs["csf_odf"] = op.abspath(self._gen_filename("csf_odf"))
+
+        for tissue_desc, tissue_type in (
+            ("White matter", "wm"),
+            ("Gray matter", "gm"),
+            ("Cerebrospinal fluid", "csf"),
+        ):
+            response_function = getattr(self.inputs, tissue_type + "_txt")
+            response_function_data = response_function_to_bids(response_function)
+
+            outputs[tissue_type + "_odf_metadata"] = {
+                "Model": {
+                    "Description": "Multi-Shell Multi-Tissue (MSMT) "
+                    "Constrained Spherical Deconvolution (CSD)",
+                    "URL": "https://mrtrix.readthedocs.io/en/latest/"
+                    "constrained_spherical_deconvolution/multi_shell_multi_tissue_csd.html",
```

Comment:
```suggestion
                    "Description": (
                        "Multi-Shell Multi-Tissue (MSMT) "
                        "Constrained Spherical Deconvolution (CSD)"
                    ),
                    "URL": (
                        "https://mrtrix.readthedocs.io/en/latest/"
                        "constrained_spherical_deconvolution/multi_shell_multi_tissue_csd.html"
                    ),
```
I find having parens makes it easier to read.